app/views/ships/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h1>New Spaceship</h1>
+<%= form_with(model: @ship, local: true) do |form| %>
+  <div>
+    <%= form.label :name %>
+    <%= form.text_field :name %>
+  </div>
+  <div>
+    <%= form.label :description %>
+    <%= form.text_area :description %>
+  </div>
+  <div>
+    <%= form.label :price_per_day %>
+    <%= form.number_field :price_per_day %>
+  </div>
+  <div>
+    <%= form.label :available_from %>
+    <%= form.date_select :available_from %>
+  </div>
+  <div>
+    <%= form.label :available_to %>
+    <%= form.date_select :available_to %>
+  </div>
```

Comment:
we will also need the :category input, which will come as a dropdown, and where it would be nice to be able to select more than one. And the :capacity input too.